marathon_acme/tests/test_clients.py

```diff
@@ -665,6 +666,9 @@ def test_get_events(self):
         request.finish()
         yield d
 
+        # Expect request.finish() to result in a logged failure
+        flush_logged_errors(ResponseDone)
```

Comment:
Does `flush_logged_errors()` return the errors it flushed? I think the version in Twisted does that.